docs/changelog.txt

```diff
@@ -60,6 +60,7 @@ Template for new versions:
 
 ## Misc Improvements
 - `dig`: `digtype` command now has options to choose between designating only visible tiles or hidden tiles, as well as "auto" dig mode. Z-level options adjusted to allow choosing z-levels above, below, or the same as the cursor.
+- ``dfhack.TranslateName``: protect against bad language indices added by mods
```

Comment:
I would put this under `Fixes` since it's a fix for a crash. Maybe something like:

```suggestion
- ``dfhack.TranslateName()``: fixed crash on certain invalid names, which affected `warn-starving`
```

Reply:
Done